SampleGoGrpc/Math.fbs

```diff
@@ -0,0 +1,16 @@
+namespace SimpleMath;
```

Comment:
SampleGoGrpc was left only for this PR, I will create tests based on monster fbs. This folder will be removed